testing/scorer/scorer-strategies/semantic-similarity/pom.xml

```diff
@@ -17,12 +17,13 @@
         <dependency>
             <groupId>dev.langchain4j</groupId>
             <artifactId>langchain4j-embeddings-bge-small-en-v15</artifactId>
+            <version>${langchain4j-embeddings.version}</version>
```

Comment:
This is a problem when I bump `langchain4j.version` to a version different than `langchain4j-embeddings.version` (for development purposes) - because then the version of this artifact is taken from the langchain4j bom instead of the `langchain4j-embeddings.version` property, making it inconsistent and forcing me to compile the `embeddings` repo as well to be able to build